src/panels/lovelace/views/get-view.ts

```diff
@@ -0,0 +1,31 @@
+import { LovelaceViewElement } from "../../../data/lovelace";
+
+import "./masonry-view";
+
+const LAYOUT_PREFIX = "ll-view-";
+const CUSTOM_PREFIX = "custom:";
+
+const ALWAYS_LOADED_LAYOUTS = new Set(["masonry"]);
+
+const LAZY_LOAD_LAYOUTS = {
+  panel: () => import("./panel-view"),
+};
+
+export const getLovelaceViewElement = (name: string): LovelaceViewElement => {
```

Comment:
```suggestion
export const createLovelaceViewElement = (name: string): LovelaceViewElement => {
```

Reply:
Use `createLovelaceElement` to create the view element

Reply:
hmm to do this I would need to change the Tags for the views up a bit

instead of `ll-view-masonry` changed to `hui-masonry-view`

Reply:
Making that change does work and I am good to change it. I was just originally following the naming from Paulus's strategy PR. (its a draft and meant to be changed. using the existing functions make sense)

Reply:
Also, I will need to change the config variable for the view from Layout to Type in order to use the existing functions

Reply:
Or. I can add in a new argument for the function that takes a config variable to look at (defaults to Type)

`config['configVar']` instead of `config.type` in `tryCreateLovelaceElement`

Reply:
I think type is fine for view? Having it the same throughout Lovelace makes sense?

Reply:
another problem I am running into

Our current methods have to run setConfig.

My current implementation doesn't need setup config as it doesn't need to know the view config to layout the cards. `hui-view` creates the card elements and sends them to the view. This should make it easier for Custom Devs since they wont need to import the create lovelace element stuff as well as the ll rebuild for the cards. Should I just create a blank setup config function on each view. So that custom devs can have it if they want It?

Reply:
Yes, we need to be able to pass config and to validate it.